### PR TITLE
[MWPW-135420] Undesirable links in search engine results

### PR DESCRIPTION
**Pull request by chadsunvice**
❗ Do not merge before Milo [PR#1189](https://github.com/adobecom/milo/pull/1189) is merged ❗ 

PR regards Milo ticket **[MWPW-133639](https://jira.corp.adobe.com/browse/MWPW-133639): Undesirable links in search engine results**.

**Contains:** 
* changes to selectors affected by ticket (IMS login checks would be affected otherwise)
* moved Playwright package to dependencies (linter errors in files because Playwright lib was moved to `devDependencies`)

❗ Do not merge before Milo [PR#1189](https://github.com/adobecom/milo/pull/1189) is merged ❗

### CODE REVIEW

**Comment by chadsunvice** [comment]
Because Playwright lib was moved (probably by mistake) to `devDependencies`, VSCode is showing multiple aggravating warnings for import statements. eg: `import { expect } from '@playwright/test';`

The PR contains the updated `package-lock.json` with the Playwright library moved to the `dependencies` section.

Screenshot 2023-08-31 at 23 30 24

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
   },
   "homepage": "https://github.com/adobecom/nala#readme",
   "dependencies": {
+    "@playwright/test": "^1.37",
```
Comment (amauch-adobe) [inline package.json:28]: Thank you for moving this to here, I also dislike the annoying linting errors.  We all should remember when we update the playwright version it needs to stay under the dependencies section and not developer dependencies.

**Review by Dli3 — APPROVED** [review]

File: selectors/feds/feds.header.page.js [hunk 2]
```diff
@@ -16,6 +16,7 @@ export default class FedsHeader {
 
     // GNAV action selectors:
     this.signInLabel = page.locator('a.feds-signIn, a.gnav-signin');
+    this.signInButton = page.locator('button[qa="gnav-ims-sign-in"]');
```
Comment (mokimo) [inline selectors/feds/feds.header.page.js:19]: th qa attribute didnt make it, adding https://github.com/adobecom/milo/pull/1189/files#diff-e52c8bf94fa988123bb9d85ff55fb1ce40556d1d6938bd0f6401f37969354023R42 should work

Comment (chadsunvice) [reply selectors/feds/feds.header.page.js:19]: Thanks @mokimo, I'll check and make the required changes.

Comment (chadsunvice) [reply selectors/feds/feds.header.page.js]: Fixed!

**Pull request merged**